### DISCUSSION TITLE
Getting only spot reading using LibrePro sensors with NFC scan

### DISCUSSION THREAD

**Discussion by T1D-Dad** [NightscoutFoundation/xDrip#3713] [Bug reports]
Hi,
I have discussed this in the [xdrip facebook page](https://www.facebook.com/share/p/EAooyW24d6DkREaL/)

Problem statement - Only spot readings are appearing in xdrip, past trends between scans are not appearing.

Impact - There are 100s of Type1 in our T1-Community, who couldn't use xdrip, as it doesnt support NFC only. Few like me use with miaomiao3 and nightscout. LibrePro sensor is mostly used in India. Please support.

![WhatsApp Image 2024-10-12 at 17 03 26](https://github.com/user-attachments/assets/141656e5-1d62-45c6-8a07-e82a529da315)
![WhatsApp Image 2024-10-12 at 17 03 25](https://github.com/user-attachments/assets/e892edb2-2c4b-4f40-a531-689ac098c568)
![WhatsApp Image 2024-10-12 at 17 03 25(1)](https://github.com/user-attachments/assets/6e782155-38d1-4b13-b4b8-e61082c6a730)

sensor - Libre Pro
Method - NFC scan (No bluetooth)
latest Android phones with NFC used.
xdrip + version - attached
event log - attached
Libre trend - attached

**Comment by Navid200** [comment]
@tzachi-dar @keencave Would either of you or both have any advice please?

**Reply by T1D-Dad** [reply]
Please help in resolving it @tzachi-dar @keencave

**Reply by keencave** [reply]
I dont have a Libre Pro available.
